<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>videos re-encoder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main_resources.qrc">
    <normaloff>:/res/image/app_icon.ico</normaloff>:/res/image/app_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_outer">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_clear">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 0, 0);
color: white;</string>
          </property>
          <property name="text">
           <string>clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_sort">
          <property name="text">
           <string>sort</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_files"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_remove_item">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="main_resources.qrc">
            <normaloff>:/res/image/resources/minus.png</normaloff>:/res/image/resources/minus.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_inside_collapsible">
        <item>
         <widget class="QComboBox" name="comboBox_preset">
          <item>
           <property name="text">
            <string>custom</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="VideoInfoWidget" name="videoInfoWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_diropen">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_output_dir"/>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_output_filename">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTimeEdit" name="timeEdit_item_length">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="displayFormat">
           <string>H'h'mm'm'ss's'</string>
          </property>
          <property name="time">
           <time>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
           </time>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>H'h'mm'm'ss's'</string>
      </property>
      <property name="time">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionopen"/>
   </widget>
   <widget class="QMenu" name="menusettings">
    <property name="title">
     <string>settings</string>
    </property>
    <addaction name="actionsavefile_name_generator"/>
    <addaction name="actioneffective_period_of_cache"/>
    <addaction name="actiondefault_preset"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menusettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionenable_tracking_of_current_time_slider">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>enable tracking of current time slider</string>
   </property>
  </action>
  <action name="actiondefault_extractor">
   <property name="text">
    <string>default extractor</string>
   </property>
  </action>
  <action name="actionsavefile_name_generator">
   <property name="text">
    <string>savefile name generator</string>
   </property>
  </action>
  <action name="actioneffective_period_of_cache">
   <property name="text">
    <string>effective period of cache</string>
   </property>
  </action>
  <action name="actiondefault_video_info">
   <property name="text">
    <string>default video info</string>
   </property>
  </action>
  <action name="actioncollapsible_section_animation_duration">
   <property name="text">
    <string>collapsible section animation duration</string>
   </property>
  </action>
  <action name="actionanimation_duration_of_collapsible_section">
   <property name="text">
    <string>animation duration of collapsible section</string>
   </property>
  </action>
  <action name="actiondefault_preset">
   <property name="text">
    <string>default preset</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoInfoWidget</class>
   <extends>QWidget</extends>
   <header>videoinfowidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="main_resources.qrc"/>
 </resources>
 <connections/>
</ui>
